mods/ra/maps/fort-lonestar/map.yaml

```diff
@@ -546,12 +533,28 @@ Rules:
 		GiveUnitCrateAction@e7:
 			Units: e7
 			SelectionShares: 10
+		GrantUpgradeCrateAction@ironcurtain:
+			SelectionShares: 10
+			Effect: invuln
+			Notification: ironcur9.aud
+			Upgrades: invulnerability
+			Duration: 1200
+		ExplodeCrateAction@bigboom:
+			Weapon: SCUD
+			SelectionShares: 5
+		GiveMcvCrateAction:
+			SelectionShares: 0
+			NoBaseSelectionShares: 1000
+			Units: mobiletent
```

Comment:
Needs `ValidFactions` to avoid:

![image](https://cloud.githubusercontent.com/assets/756669/12704398/6fd0b816-c85a-11e5-82f2-9935fa5ccbdc.png)